agent/acl_endpoint_test.go

```diff
@@ -323,7 +311,7 @@ func TestACL_HTTP(t *testing.T) {
 			policies, ok := raw.(structs.ACLPolicyListStubs)
 			require.True(t, ok)
 
-			// 2 we just created + global management
+			// 3 we just created + global management
```

Comment:
Did you mean to change this?

Reply:
At first I put the new tests above this and had to change it but them I moved them and forgot to reupdate the comment.